### DISCUSSION TITLE
Welcome to MpsAgent Discussions!

### DISCUSSION THREAD

**Discussion by dgkanatsios** [PlayFab/MpsAgent#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ToniaSanzo** [comment]
I think the build directions should be updated in the README.md:

```
git clone https://github.com/PlayFab/MpsAgent.git
cd ./MpsAgent/LocalMultiplayerAgent 
# replace the LocalMultiplayerAgentPublishFolder with the folder of your choice
dotnet publish --runtime win-x64 -c Release -o LocalMultiplayerAgentPublishFolder -p:PublishSingleFile=true -p:PublishTrimmed=true --self-contained true
# you can read here about .NET publish CLI options https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish
```

**Reply by dgkanatsios** [reply]
Ah, great catch, thanks! Would you like to do a PR for this?
